ransack.gemspec

```diff
@@ -6,8 +6,8 @@ Gem::Specification.new do |s|
   s.name        = "ransack"
   s.version     = Ransack::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Ernie Miller", "Ryan Bigg", "Jon Atack"]
-  s.email       = ["ernie@erniemiller.org", "radarlistener@gmail.com", "jonnyatack@gmail.com"]
```

Comment:
My understanding of `authors` is that it represents the people who wrote a substantial portion of the library. I could certainly be wrong, though.

On the other extreme, ActiveAdmin only lists Greg Bell as an author:

<img width="387" alt="screen shot 2017-12-31 at 2 58 10 pm" src="https://user-images.githubusercontent.com/688886/34464179-0e9f2902-ee3b-11e7-9962-e885dac5cdec.png">

Reply:
Any chance the code could be tested @seanlinsley ?

Reply:
Agreed. Do not change authors. I wonder even if it's required to change anything since ~1.3 means 1.3.2 too

Reply:
Unless there's a reason to prevent applications from using 1.3.0 and 1.3.1, @bbonislawski appears to be right.

Reply:
Given there's no "maintainers" section, I'm fine with adding an author, personally.

Reply:
Thanks @ernie